docs/references/phasing.tex

```diff
@@ -1,111 +1,179 @@
+Taken all together, the projected baseline vectors, $\mathbf{r}^{\prime}$, can be represented as:
+\begin{equation}
+\mathbf{r}^{\prime} = \mathcal{R}_{\phi_{\rm FPA},x} \mathcal{R}_{\delta_{\rm PC}, y} \mathcal{R}_{\textrm{GHA}, z} \mathbf{r}_{\rm ant}
+\end{equation}
+\subsection{Projection}\label{ssec:projection}
+Once the $uvw$-coordinates are calculated, delay compensation for the $w$-projected distance is relatively straightforward, where the following per-visibility phase correction, $\phi^{\prime}$ is applied:
+\begin{equation}\label{eqn:projection}
+\phi^{\prime} = e^{-2i\pi\Delta w / \lambda_{\rm obs}},
+\end{equation}
+where $\Delta$w is the difference in $w$-projected distance (between the new and old phase center) for a given visibility, and $\lambda_{\rm obs}$ is the wavelength of the frequency channel the visibility belongs to. Note that for unprojected data, no phase term has been applied, so $\Delta w$ is the full distance from the baseline pair to the plane. Note that delay corrections are applied using the observed frequency, as stored in \texttt{UVData.freq\_array}, versus the rest-frame frequency of the object in question, which is not accounted for by pyuvdata.
 
-In addition, there are some discrepancies in the LST calculations done in the MWA code and in \texttt{astropy}. The MWA code uses the mean LST to calculate the RA of zenith in the True Equator and Equinox frame (TEE, also referred to as the current epoch). The mean LST calculated by \texttt{astropy} differs from the mean LST calculated in the MWA code by about 4 arcseconds at the particular time used for testing (mjd = 55780.1). Furthermore, the authors believe that the LST used in this calculation should actually be the apparent LST (which corrects for nutation in addition to precession). The apparent LST calculated by \texttt{astropy} differs from the mean LST calculated in the MWA code by about 12 arceseconds at the same testing time. Note that the LST is not used in \texttt{pyuvdata}'s phasing code, instead all the precession and nutation corrections are applied in the \texttt{astropy} frame conversions.
+One final note -- all of the above assumes ``far-field'' objects, where in incoming wavefront can safely be presumed to be planar. Phasing for near-field objects (where the wavefront needs to be treated as spherical) is not yet supported, though exploratory development work is presently underway to support such phase centers.
```

Comment:
where in incoming wavefront -> where the incoming wavefront

Reply:
Fixed.